### PR TITLE
output entities migration ready + fake data for outputs

### PR DESCRIPTION
**Pull request by aagm**
Migration + fake data for outputs

### CODE REVIEW

**Review by kgajowy — CHANGES_REQUESTED** [review]
1. Failing tests 
2. We shall keep entities definitions (.entity.ts) accurate at least for columns

File: api/apps/geoprocessing/src/migrations/geoprocessing/1624890503611-marxanOutputEntities.ts [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        ADD CONSTRAINT scenario_pufk FOREIGN KEY (scenario_pu_id)
+            REFERENCES scenarios_pu_data (id)
+            ON DELETE CASCADE;
+      ALTER TABLE output_scenarios_pu_data
+        DROP COLUMN run_id;
+      ALTER TABLE output_scenarios_pu_data
+        ADD COLUMN run_id int;
+
+      ALTER TABLE output_scenarios_pu_data
+        ADD CONSTRAINT value_out_chk CHECK (value = 0 or value = 1);
```
Comment (hotzevzl) [inline api/apps/geoprocessing/src/migrations/geoprocessing/1624890503611-marxanOutputEntities.ts:48]: do we still need `value` to be `double precision` if we constrain it to 0 or 1?

**Review by hotzevzl — COMMENTED** [review]
thanks @aagm, the data parts look ok, though as Kamil noted we need to sync the entities.

**Review by kgajowy — APPROVED** [review]

File: api/apps/geoprocessing/src/migrations/geoprocessing/1624890503611-marxanOutputEntities.ts [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        ADD COLUMN scenario_pu_id uuid NOT NULL;
+
+      ALTER TABLE output_scenarios_pu_data
+        ADD CONSTRAINT scenario_pufk FOREIGN KEY (scenario_pu_id)
+            REFERENCES scenarios_pu_data (id)
+            ON DELETE CASCADE;
+      ALTER TABLE output_scenarios_pu_data
+        DROP COLUMN run_id;
+      ALTER TABLE output_scenarios_pu_data
+        ADD COLUMN run_id int;
```
[the remark this answers is not in the document]
Comment (hotzevzl) [reply api/apps/geoprocessing/src/migrations/geoprocessing/1624890503611-marxanOutputEntities.ts:47]: we're taking care of this in https://vizzuality.atlassian.net/browse/MARXAN-498

**Pull request merged**